components/function-controller/README.md

```diff
@@ -1,184 +1,39 @@
-
-    ```bash
-    reg_username=<container registry username>
-    reg_password=<container registry password>
-
-    cat <<EOF | kubectl -n ${FN_NAMESPACE} apply -f -
-    ---
-    apiVersion: v1
-    kind: Secret
-    type: kubernetes.io/basic-auth
-    metadata:
-      name: registry-credentials
-      annotations:
-        tekton.dev/docker-0: ${FN_REGISTRY}
-    stringData:
-      username: ${reg_username}
-      password: ${reg_password}
-    ---
-    apiVersion: v1
-    kind: ServiceAccount
-    metadata:
-      name: function-controller-build
-    secrets:
-    - name: registry-credentials
-    EOF
-    ```
-
-### Deploy the controller
-
-To deploy the Function Controller to the `serverless-system` Namespace, run:
-
-```bash
-make deploy
-```
-
-This runs [ko](https://github.com/google/ko) to build your image and push it to the container repository set in the **KO_DOCKER_REPO** environment variable.
-
-## Usage
-
-### Run tests
-
-Use the following `make` target to test the deployed Function Controller.
-
-```bash
-make test
-```
-
-### Create a sample Hello World Function
-
-Follow the steps below to create a sample Function.
-
-1. Apply the following Function manifest:
-
-    ```bash
-    kubectl -n ${FN_NAMESPACE} apply -f config/samples/serverless_v1alpha1_function.yaml
-    ```
-
-2. Ensure the Function was created:
-
-    ```bash
-    kubectl -n ${FN_NAMESPACE} get functions
-    ```
-
-3. Check the status of the build:
-
-    ```bash
-    kubectl -n ${FN_NAMESPACE} get taskruns.tekton.dev
-    ```
-
-4. Check the status of the Knative Serving service:
-
-    ```bash
-    kubectl -n ${FN_NAMESPACE} get services.serving.knative.dev
-    ```
-
-5. Access the Function:
-
-    <div tabs name="installation">
-
-      <details>
-      <summary>Minikube</summary>
-
-      ```bash
-      FN_DOMAIN="$(kubectl -n ${FN_NAMESPACE} get ksvc demo --output 'jsonpath={.status.url}' | sed -e 's/http\([s]\)*:[/][/]//')"
-      FN_PORT="$(kubectl get svc istio-ingressgateway -n istio-system --output 'jsonpath={.spec.ports[?(@.port==80)].nodePort}')"
-      curl -v -H "Host: ${FN_DOMAIN}" http://$(minikube ip):${FN_PORT}
-      ```
-      </details>
-
-      <details>
-      <summary>Remote cluster</summary>
-
-      ```bash
-      FN_DOMAIN="$(kubectl -n ${FN_NAMESPACE} get ksvc demo --output 'jsonpath={.status.url}' | sed -e 's/http\([s]\)*:[/][/]//')"
-      FN_INGRESS="$(kubectl get svc istio-ingressgateway -n istio-system --output 'jsonpath={.status.loadBalancer.ingress[0].ip}')"
-      curl -kD- -H "Host: ${FN_DOMAIN}" "http://${FN_INGRESS}"   
-      ```
-      </details>
-
-    </div>
+> **CAUTION:** Functions works only in the `serverless` Namespace that is created as part of the `function-controller` chart. To avoid authentication issues, add a trusted certificate to your cluster. Read [here](https://kyma-project.io/docs/#installation-install-kyma-with-your-own-domain-generate-the-tls-certificate) how to obtain one.
```

Comment:
```suggestion
> **CAUTION:** Functions work only in the `serverless` Namespace that is created as part of the `function-controller` chart. To avoid authentication issues, add a trusted certificate to your cluster. Read [here](https://kyma-project.io/docs/#installation-install-kyma-with-your-own-domain-generate-the-tls-certificate) how to obtain one.
```

Reply:
I would put it in the Overview, it would be easily seen then.